pkg/network/usm/testutil/grpc/client.go

```diff
@@ -46,8 +46,8 @@ func (c *Client) HandleStream(ctx context.Context, numberOfMessages int32) error
 		input[i] = i + 1
 	}
 
-	rand.Seed(time.Now().UnixNano())
-	rand.Shuffle(len(input), func(i, j int) { input[i], input[j] = input[j], input[i] })
+	r := rand.New(rand.NewSource(time.Now().UnixNano()))
+	r.Shuffle(len(input), func(i, j int) { input[i], input[j] = input[j], input[i] })
```

Comment:
The default generator of the `rand` package is now automatically randomly seeded at program startup.
This should be equivalent to:

```suggestion
	rand.Shuffle(len(input), func(i, j int) { input[i], input[j] = input[j], input[i] })
```

Reply:
LGTM